lib/sdt_alloc.bpf.c

```diff
@@ -231,20 +260,20 @@ sdt_desc_t *sdt_alloc_chunk(struct sdt_alloc_stack __arena *stack)
 	return out;
 }
 
-static SDT_TASK_FN_ATTRS int sdt_pool_set_size(struct sdt_pool *pool, __u64 data_size)
+static SDT_TASK_FN_ATTRS int sdt_pool_set_size(struct sdt_pool *pool, __u64 data_size, __u64 alloc_size)
```

Comment:
Maybe sth like `nr_pages` instead of `alloc_size`? The units can be confusing if len/size are used for page count.